Src/AutoNSubstitute/AutoNSubstitute.csproj

```diff
@@ -26,7 +26,7 @@
     <WarningLevel>4</WarningLevel>
     <CodeAnalysisRuleSet>AutoNSubstitute.ruleset</CodeAnalysisRuleSet>
     <DocumentationFile>bin\Release\Ploeh.AutoFixture.AutoNSubstitute.XML</DocumentationFile>
-    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <RunCodeAnalysis>false</RunCodeAnalysis>

```

Comment:
It's not even in Release mode, but in Debug mode, I see now.

AFAICT, this was added in 627c6d85b9482bbd9424561b06e491985de52980, and went unnoticed for 2½ years :flushed:

Good catch :+1: